package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@scality/core-ui",
-  "version": "0.40.0",
+  "version": "0.41.0",
```

Comment:
Hello @ognyan-kostadinov,
It's great you always remember to bump the package version within the PR. 

Just FYI, we are trying to automate as much as possible the manual process to release a new version.

We deployed the **post-release** GitHub action to help us bump the version which is the **next release version**.

And we've already succeeded to bump the version after the previous release: **0.39.8**. 

So actually the version in the package.json now is the version we are going to release ;)